modules/tinymce/src/plugins/help/test/ts/browser/CustomTabsTest.ts

```diff
@@ -1,149 +1,152 @@
-import Editor from '../../../../../core/main/ts/api/Editor';
-import Theme from '../../../../../themes/silver/main/ts/Theme';
+import { Html, SugarDocument } from '@ephox/sugar';
+import { assert } from 'chai';
 
-UnitTest.asynctest('Custom Help Tabs test', (success, failure) => {
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/help/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
 
-  HelpPlugin();
-  Theme();
-
-  const doc = SugarElement.fromDom(document);
-
-  const compareTabNames = (expectedNames: string[]) => Chain.op((editor: Editor) => {
-    editor.execCommand('mceHelp');
-    const actualTabs = UiFinder.findAllIn(doc, 'div.tox-dialog__body-nav-item.tox-tab');
-    const actualNames: string[] = Arr.map(actualTabs, (tab) => Html.get(tab));
-    Arr.map(expectedNames, (x, i) => assert.eq(x, actualNames[i], `Tab names did not match. Expected: ${expectedNames}. Actual: ${actualNames}`));
+describe('browser.tinymce.plugins.help.CustomTabsTest', () => {
+  before(() => {
+    Plugin();
+    Theme();
   });
 
-  const makeStep = (config: Object, expectedTabNames: string[]) => Chain.asStep({}, [
-    McEditor.cFromSettings(config),
-    NamedChain.asChain([
-      NamedChain.direct(NamedChain.inputName(), Chain.identity, 'editor'),
-      NamedChain.read('editor', compareTabNames(expectedTabNames)),
-      NamedChain.output('editor')
-    ]),
-    McEditor.cRemove
-  ]);
-
-  Pipeline.async({}, [
-    Logger.t('Default help dialog', makeStep({
-      plugins: 'help',
-      toolbar: 'help',
-      theme: 'silver',
-      base_url: '/project/tinymce/js/tinymce'
-    }, [ 'Handy Shortcuts', 'Keyboard Navigation', 'Plugins', 'Version' ])),
+  const compareTabNames = (editor: Editor, expectedNames: string[]) => {
+    editor.execCommand('mceHelp');
+    const actualTabs = UiFinder.findAllIn(SugarDocument.getDocument(), 'div.tox-dialog__body-nav-item.tox-tab');
+    const actualNames = Arr.map(actualTabs, (tab) => Html.get(tab));
+    Arr.map(expectedNames, (x, i) => {
+      assert.equal(x, actualNames[i], `Tab names did not match. Expected: ${expectedNames}. Actual: ${actualNames}`);
+    });
+  };
 
-    Logger.t('Test help_tabs with pre-registered and new tabs', makeStep({
-      plugins: 'help',
-      toolbar: 'help',
-      theme: 'silver',
-      base_url: '/project/tinymce/js/tinymce',
-      help_tabs: [
-        'shortcuts',
-        'plugins',
-        {
-          name: 'versions', // this will override the default versions tab
-          title: 'Version',
-          items: [{
-            type: 'htmlpanel',
-            html: '<p>This is a custom version panel...</p>'
-          }]
-        },
-        {
-          name: 'extraTab1',
-          title: 'Extra1',
-          items: [{
-            type: 'htmlpanel',
-            html: '<p>This is an extra tab</p>'
-          }]
-        }
-      ]
-    }, [ 'Handy Shortcuts', 'Plugins', 'Version', 'Extra1' ])),
+  Arr.each([
```

Comment:
Can we declare this array above the Arr.each call, possibly with an interface explaining the pieces involved (unsure on that one).

Reply:
Why, what will that achieve? This IMO is quite neat and tidy, plus also saves a fair bit of boilerplate, so I'd like a really good reason as to why we should change it.

Reply:
As discussed on slack, I've split this up into individual tests as we're okay with using `Arr.each` for `describe` but we want to avoid it for `it` as it makes it hard to see what's being tested and also to debug individual tests.